components/LatestEvents.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+    <div class="events-container">
+        <div class="top-heading">
+            <h1>
+                LATEST EVENTS
+            </h1>
+        </div>
+        <div class="latest-events">
+          <div class= "latest-events__main">
+            <div class="main-box-container">
+                <div class="main-box">
+                    <div class="main-box__image-box">
+                        <img :src="eventsList[0].image===null?'':eventsList[0].image.url"></img>
+                    </div>
+                    <div class="main-box__title">
+                        <h4>{{eventsList[0].title}}</h4>
+                    </div>
+                    <div class="main-box__dates">
+                        <span>
+                            {{formatDate(eventsList[0].startDate)}} - {{formatDate(eventsList[0].endDate)}}
+                        </span>
+                    </div>
+                    <div class="main-box__blurb">
+                        <span>
+                            {{eventsList[0].blurb}}
+                        </span>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class= "latest-events__upcoming">
+            <div class="upcoming-box">
+                <h2>UPCOMING EVENTS</h2>
+                <div class="upcoming-box__grid" v-for="(eventsItem, index) in eventsList">
+                    <div :style="index === 0? itemStyle : '' " class="grid-item">
+                        <span>{{formatMonth(eventsItem.startDate).toUpperCase()}}</span>
+                        <span class="day">{{formatDay(eventsItem.startDate)}}</span>
+                    </div>   
+                    <div :style="index === 0? itemStyle : '' " class="grid-item">
+                        <span>
+                            {{eventsItem.title}}
+                        </span>
+                    </div>               
+                </div>
+                <div class="view-all">
+                    <a href="news-and-events/events/">VIEW ALL EVENTS</a>
```

Comment:
Any link internal to the nuxt site should use the following component instead:
https://nuxtjs.org/docs/features/nuxt-components/#the-nuxtlink-component

It will be rendered as <a> at the end so the style should stay.